src/execute-pr-actions.ts

```diff
@@ -1,4 +1,4 @@
-import { PR as PRQueryResult, PR_repository_pullRequest } from "./schema/PR";
+import { PR as PRQueryResult, PR_repository_pullRequest } from "./queries/schema/PR";
```

Comment:
Unrelated change: we had autogenerated schemas in two places; I removed the outdated ones.